request/request.go

```diff
@@ -157,3 +159,9 @@ func GetHTTPHeader(header *BaseHeader) http.Header {
 
 	return httpHeader
 }
+
+func AddExtraHeaders(header http.Header) http.Header {
+	header.Add("LibraryVersion", constants.GoExtraHeaders.LibraryVersion)
+	header.Add("LibraryPlatform", constants.GoExtraHeaders.LibraryPlatform)
```

Comment:
Move these calls to the caller side.

Reply:
Fixed: https://github.com/dapi-co/dapi-go/pull/10/commits/562ba8f9aa1f1d67b8faee8f7dd336a867e69e25